src/app/helptext/system/cloudcredentials.ts

```diff
@@ -87,34 +87,13 @@ export const helptext_system_cloudcredentials = {
   },
 
   account_b2: {
-    placeholder: T("Account ID or Application Key ID"),
-    tooltip: T(
-      'Alphanumeric <a\
- href="https://www.backblaze.com/b2/cloud-storage.html"\
- target="_blank">Backblaze B2</a> ID. Find an Account ID\
- or applicationKeyID by logging in to the account,\
- clicking <i>Buckets</i>, and clicking\
- <i>Show Account ID and Application Key</i>. Enter the\
- <i>Account ID</i> to associate the entire account or\
- generate a new <i>Application Key</i>. The <i>keyID</i>\
- replaces the Account ID and the key string is used in\
- place of the <i>Master Application Key</i>.'
-    )
+    placeholder: T("Key ID"),
+    tooltip: T(''),
   },
 
   key_b2: {
-    placeholder: T("Master Application Key or Application Key"),
-    tooltip: T(
-      'Backblaze B2 Application Key. Log in to\
- the B2 account and generate a key on the Buckets\
- page. <a\
- href="https://help.backblaze.com/hc/en-us/articles/224991568-Where-can-I-find-my-Account-ID-and-Application-Key-"\
- target="_blank">Generating a new Master Application Key</a>\
- will invalidate the existing Master key and require\
- updating this field. Using a limited permissions\
- Application Key also requires changing the\
- <i>Account ID</i> to the new <i>keyID</i>.'
-    )
+    placeholder: T("Application Key"),
+    tooltip: T(''),
```

Comment:
`<a href="https://www.backblaze.com/b2/cloud-storage.html" target="_blank">Backblaze B2</a>` Application Key. To generate a new application key, log in to the Backblaze account, go to the `<i>App Keys</i>` page, and add a new application key. Copy the `<i>applicationKey</i>` string to this field.